docs/omero-release-process.rst

```diff
@@ -14,19 +14,26 @@ The release process uses GitHub actions, make sure that the actions are active b
 Register CVE
 ^^^^^^^^^^^^
 
-To be added
+As soon as a CVE is identified, create a security advisory on `GitHub <https://github.com/ome/openmicroscopy/security/advisories>`_.
+The work to fix the CVE will be done using the private copy of `ome/openmicroscopy <https://github.com/ome/openmicroscopy/>`_ and the private copies of the Java components.
```

Comment:
What about OMERO.web vulnerabilities? Should these be captured by this document? Are the advisories expected to be drafted on https://github.com/ome/omero-web/security/advisories?